assets/js/components/user-input/UserInputSelectOptions.js

```diff
@@ -113,11 +140,16 @@ export default function UserInputSelectOptions( { slug, options, max } ) {
 						{ __( 'Other:', 'google-site-kit' ) }
 					</ListComponent>
 
-					<TextField>
+					<TextField
+						label={ __( 'Type your own answer', 'google-site-kit' ) }
```

Comment:
This is a bit friendler for screen-readers than "Other", especially when focus is changed.